goDcurl/src/pow_dcurl/pow_dcurl_test.go

```diff
@@ -0,0 +1,38 @@
+
+	mwm := 14
+	t.Log("Testing an 8 threads dcurl pow with 14 as mwm")
+	nonce, err := DcurlProofOfWork(rawTx, mwm, 8)
+	if err == nil {
+		t.Log("dcurl PoW successed")
+	} else {
+		t.Error("dcurl PoW failed")
+	}
+
+	if len(nonce) == 27 {
+		t.Log("Correct length of nonce")
+	} else {
+		t.Error("Wrong length of nonce")
+	}
+	rawTx = rawTx[:len(rawTx)-NonceTrinarySize/3] + nonce
+	hashedTrytes := curl.MustHashTrytes(rawTx)
+
+	t.Log("hashed Trytes:", hashedTrytes)
+	if guards.IsTransactionHashWithMWM(hashedTrytes, 10) {
+		t.Log("Correctly hash with mwm")
+	} else {
+		t.Error("Wrong hash trytes")
```

Comment:
Unify the log and error message.

Reply:
Thanks!
Use "Correct hash with mwm" and "Wrong hash with mwm" now.